pioneer.py

```diff
@@ -104,42 +106,88 @@ def main():
                 stop_game = True
             
             #### Hero Movement
+            
```

Comment:
This may help with edge of screen detection:
 if keys_down["up"]:
        if player.y > 0:
            player.y -= player.speed
    elif keys_down["down"]:
        if player.y < 660:
            player.y += player.speed
    if keys_down["left"]:
        if player.x > 0:
            player.x -= player.speed
    elif keys_down["right"]:
        if player.x < 1270:
            player.x += player.speed